packages/shared/ReactFeatureFlags.js

```diff
@@ -176,4 +176,4 @@ export const allowConcurrentByDefault = false;
 
 export const enablePersistentOffscreenHostContainer = false;
 
-export const consoleManagedByDevToolsDuringStrictMode = false;
+export const consoleManagedByDevToolsDuringStrictMode = true;
```

Comment:
I'd expect this to be true because it's OSS web React, which _is_ managed by DevTools

Reply:
This was an oversight from the previous PR. Thanks for catching.